internal/service/iam/policy.go

```diff
@@ -101,6 +102,11 @@ func resourcePolicy() *schema.Resource {
 			},
 			names.AttrTags:    tftags.TagsSchema(),
 			names.AttrTagsAll: tftags.TagsSchemaComputed(),
+			"delay_after_policy_creation_in_ms": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  -1,
+			},
```

Comment:
The default value can be removed, and below the check should be `d.GetOk`. The `ok` value will be `false` if there is no value set.